src/java/com/sarality/app/view/list/EmptyListViewRenderer.java

```diff
@@ -32,9 +32,14 @@ void render(ListView listView) {
 
   void render(LinearLayout layout, boolean isListEmpty) {
     View emptyView = getViewToRender();
+
+    if (layout.findViewById(emptyView.getId()) == null) {
```

Comment:
Yes you do. How else would we show a NonScrollingListView. You just need to include an EmptyView next to it as well just like you would have done in a ListView.